pkg/controller/controller.go

```diff
@@ -374,6 +376,66 @@ func (c *CSIControllerService) ListSnapshots(context.Context, *csi.ListSnapshots
 }
 
 // ControllerExpandVolume is not implemented yet
-func (c *CSIControllerService) ControllerExpandVolume(context.Context, *csi.ControllerExpandVolumeRequest) (*csi.ControllerExpandVolumeResponse, error) {
-	return nil, status.Error(codes.Unimplemented, "not implemented yet")
+func (c *CSIControllerService) ControllerExpandVolume(ctx context.Context, req *csi.ControllerExpandVolumeRequest) (*csi.ControllerExpandVolumeResponse, error) {
+	ll := c.log.WithFields(logrus.Fields{
+		"method":   "ControllerExpandVolume",
+		"volumeID": req.GetVolumeId(),
+	})
+	ll.Infof("Processing request: %v", req)
+	var (
+		volID     = req.GetVolumeId()
+		lvg       = &lvgcrd.LVG{}
+		ctxWithID = context.WithValue(context.Background(), base.RequestUUID, volID)
+	)
+
+	if volID == "" {
+		return nil, status.Error(codes.InvalidArgument, "Volume name missing in request")
+	}
+	if req.GetVolumeCapability() == nil {
+		return nil, status.Error(codes.InvalidArgument, "Volume capabilities missing in request")
+	}
+
+	volume := c.crHelper.GetVolumeByID(volID)
```

Comment:
I think we can try to improve error handling in all chHelper methods. Could you please check how hard it will be to change `GetVolumeByID` signature to return error alongside volume? I think it will be great if we can handle "not found" and "api call error" separately.